### DISCUSSION TITLE
DKL-Cloud (16/01/2024)

### DISCUSSION THREAD

**Discussion by progettistamatto** [sapienzastudentsnetwork/basi-di-dati-2#50] [Esami]
In questa discussion a tema il progetto "DKL-Cloud", oggetto del testo di esame del 16 gennaio 2024, discuteremo di eventuali proposte di soluzione, dubbi o scelte progettuali in merito alla realizzazione del progetto in questione

**Comment by CosaFaSelect1** [comment]
Per questo esame è disponibile [[QUI]](https://github.com/CasuFrost/university-notes-Informatica/raw/main/Secondo%20Anno/Secondo%20Semestre/Basi%20di%20Dati%202/altra%20roba/DKL.pdf) una soluzione proposta da Marco Casu
